workos/resources/workos_model.py

```diff
@@ -1,19 +1,21 @@
+from typing import Any, Dict, Optional
 from typing_extensions import override
 from pydantic import BaseModel
+from pydantic.main import IncEx
 
 
 class WorkOSModel(BaseModel):
     @override
     def dict(
         self,
         *,
-        include=None,
-        exclude=None,
-        by_alias=False,
-        exclude_unset=False,
-        exclude_defaults=False,
-        exclude_none=False
-    ):
+        include: Optional[IncEx] = None,
```

Comment:
This whole method is basically an alias for `model_dump`. Is there a way to do that in `python` without all repetition?

Reply:
The winner looks to be: `dict = BaseModel.model_dump`

Reply:
Thought I could get set `dict = BaseModel.model_dump`, but since `dict()` is deprecated in the parent that causes ~strikethroughs~ everywhere dict is used.

Reply:
Booo. I wonder why they deprecated instead of aliasing. Should a `.dict()` method not accept a bunch of options? Or is `.dict()` not a python norm?

Reply:
> Or is .dict() not a python norm?

I tried to answer my own question with some googling and it turned into a rabbit hole. Going to slowly back away from this...